src/main/java/seedu/address/model/person/ParentName.java

```diff
@@ -0,0 +1,17 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class ParentName extends Name {
+    /**
```

Comment:
Maybe this could be 'name for a Parent'? Same for a few other places in this file